README.md

```diff
@@ -20,6 +20,10 @@ Run the exporter as follows:
 
 This will read the PCB layout from `example/example.kicad_pcb` and write it to `example/example.brd`.
 
+Optionally, you can also use this script to export a `.obdata` file for your boards for viewing component values and package sizes in [FlexBV][].
+
+[FlexBV]: https://pldaniels.com/flexbv
```

Comment:
So actually when I suggested mentioning FlexBV I did not realize that it's a commercial application. I think you should only refer to https://openboarddata.org in the README as that is the community project.

Reply:
Okay perfect. For the lower section should I specify that the screenshots are from the free version?

Reply:
No references to FlexBV at all, only openboarddata.org.

Reply:
So do I remove the screenshot entirely or just the mention that it is from FlexBV?

Reply:
Remove it entirely.